service/src/main/java/org/databiosphere/workspacedataservice/controller/RecordController.java

```diff
@@ -161,6 +161,20 @@ public ResponseEntity<String> createInstance(@PathVariable("instanceId") UUID in
 		return new ResponseEntity<>(HttpStatus.CREATED);
 	}
 
+	@DeleteMapping("/{instanceId}/records/{version}/{recordType}/{recordId}")
+	public ResponseEntity deleteSingleRecord(@PathVariable("instanceId") UUID instanceId,
+			@PathVariable("version") String version, @PathVariable("recordType") RecordType recordType,
+			@PathVariable("recordId") RecordId recordId) {
+		validateVersion(version);
+		String recordTypeName = recordType.getName();
+		recordDao
+				.getSingleRecord(instanceId, recordType, recordId,
+						recordDao.getRelationCols(instanceId, recordTypeName))
+				.orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Record not found"));
+		recordDao.deleteSingleRecord(instanceId, recordType.getName(), recordId.getRecordIdentifier());
+		return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+	}
+
```

Comment:
```suggestion
	@DeleteMapping("/{instanceId}/records/{version}/{recordType}/{recordId}")
	public ResponseEntity deleteSingleRecord(@PathVariable("instanceId") UUID instanceId,
			@PathVariable("version") String version, @PathVariable("recordType") RecordType recordType,
			@PathVariable("recordId") RecordId recordId) {
		validateVersion(version);
		boolean recordFound = recordDao.deleteSingleRecord(instanceId, recordType.getName(), recordId.getRecordIdentifier());
		return recordFound ? new ResponseEntity<>(HttpStatus.NO_CONTENT) : new ResponseEntity<>(HttpStatus.NOT_FOUND);
	}
```
If you elect to make my suggested changes in the DAO, this is what shakes out in the controller.